src/app/admin/invite/organization-select.tsx

```diff
@@ -0,0 +1,40 @@
+import { Text, Select, type SelectProps } from '@mantine/core'
+import { useQuery } from '@tanstack/react-query'
+import { fetchMembersForSelectAction } from '@/server/actions/member-actions'
+
+export const OrganizationSelect: React.FC<SelectProps> = (props) => {
+    const {
+        data: members = [],
```

Comment:
just in general, we're doing a lot of renaming from member -> organization, do we want to follow this general rule in code? ie here we have `OrganizationSelect` which internally uses `members` as a variable, i think this can get confusing down the line

dont think we have to do this now for time sake, but i don't want to lose track of it

also... i think this is a US vs EU thing but we're doing both `organization` and `organisation` in here 😆

Reply:
yah, I think we should discuss if we should rename `member` references in code post-mvp

Reply:
Beginning post mvp sounds like a good time frame to tackle this (and we should stick to organiZation when singular according to US English).